src/SDKs/CognitiveServices/dataPlane/Search/Microsoft.CognitiveServices.Search.Tests/Microsoft.CognitiveServices.Search.Tests.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.CognitiveServices.Search.Tests</PackageId>
+    <Description>Microsoft.CognitiveServices.Search.Tests Class Library</Description>
+    <AssemblyName>Microsoft.CognitiveServices.Search.Tests</AssemblyName>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+  </PropertyGroup>
+  
+  <PropertyGroup>    
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
+    <ProjectReference Include="..\Microsoft.CognitiveServices.Search\Microsoft.CognitiveServices.Search.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="testsettings.json">
```

Comment:
@stefanvalianu I don't see testsettings.json as part of the PR

Reply:
Hmm, i basically copied the folder structure of the Vision APIs (and files therein). Not sure where this is, but apparently I don't need it :)